bitnami/mongodb-sharded/values.yaml

```
@@ -605,6 +605,10 @@ configsvr:
     ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
     ##
     annotations: {}
+    ## @param configsvr.persistence.selector Persistent Volume selector
```

Let's sync the chart parameter definition with what we used in the rest of the catalog:
```suggestion
    ## @param configsvr.persistence.selector Selector to match an existing Persistent Volume for Config Server data PVC
```